docs/src/main/paradox/release-notes/2.0.x/google-cloud-storage.md

```diff
@@ -1,5 +1,14 @@
 # Google Cloud Storage
 
+## 2.0.0-M2
+
+- Google Cloud Storage: Add exponential backoff on server errors [#2026](https://github.com/akka/alpakka/issues/2026) by [@gabrielreid](https://github.com/gabrielreid)
+- GCS StorageObjectJson JsonFormat [#2](https://github.com/akka/alpakka/issues/2) by [@seglo](https://github.com/seglo)
```

Comment:
Something strange happened here.  Looks like it may have picked up the PR I merged to Daniele's fork.

Relevant incomplete PR: https://github.com/akka/alpakka/pull/2009

Reply:
Yeah, I noticed it, but forgot to remove it. Great find.